core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalH2OBackend.scala

```diff
@@ -71,11 +71,16 @@ class ExternalH2OBackend(val hc: H2OContext) extends SparklingBackend with Exter
       "-J", "-log_level", "-J", conf.h2oNodeLogLevel,
       "-timeout", conf.clusterStartTimeout.toString,
       "-disown",
+      "-J", "-stacktrace_collector_interval", "-J", conf.stacktraceCollectorInterval.toString,
       "-J", "-watchdog_stop_without_client",
       "-J", "-watchdog_client_connect_timeout", "-J", conf.clientConnectionTimeout.toString,
       "-J", "-watchdog_client_retry_timeout", "-J", conf.clientCheckRetryTimeout.toString
     )
 
+    if(conf.stacktraceCollectorEnabled){
+      cmdToLaunch = cmdToLaunch ++ Seq[String]("-J", "-stacktrace_collector_enabled")
```

Comment:
Do we need 2 arguments? Perhaps one argument that also sets the interval would make it simpler for everybody. I cannot image who would be happy with the default interval

Reply:
Also there is a generic mechanism to disable H2O extensions.

Reply:
Do you mean something like `-stacktrace_collector_timeout=-1` to disable the extension? I still prefer to have 2 arguments for the explicity.